ioeffect/src/main/scala/scalaz/ioeffect/IO.scala

```diff
@@ -686,6 +686,11 @@ object IO extends IOInstances {
    */
   final def sync[E, A](effect: => A): IO[E, A] = new SyncEffect(() => effect)
 
+  /**
+   * To ease migration.
+   */
+  final def apply[E, A](effect: => A): IO[E, A] = sync(effect)
```

Comment:
It was left out on purpose coz it's never clear which kind of lifting `apply` should be shortcut for

Reply:
I think it is a mistake to leave it out from a code migration point of view, and we want to make it as easy as possible for people to come back to scalaz-ioeffect from cats-effect.

It's fairly obvious (to me, at least) that an `IO` block is doing some sync work, because async is usually opt-in.

If the decision is that it shouldn't be in ZIO then there is an easy upgrade path from ioeffect to zio that can even be automated with scalafix.